lms/resources/_lti_launch.py

```diff
@@ -203,59 +243,6 @@ def js_config(self):
 
         return self._js_config
 
-    @property
-    def hypothesis_config(self):
```

Comment:
As per the comment above, if this was mostly just changed to a function `_hypothesis_config()` I think it would be easier to read:

 * This diff (not trivial to see if this function has changed)
 * The function above (as you don't have to read lots of function it might not even call)